.github/workflows/test.yaml

```diff
@@ -43,13 +43,21 @@ jobs:
       - name: Prepare build
         run: yarn e2e:pre-prod
       
-      - name: Run tests
+      - name: Run admin user tests
         run: |
-          yarn e2e:prod
+          GREP_TAGS=@adminUser TEST_USERNAME=admin yarn e2e:prod
```

Comment:
To keep things clean env vars should be passed via the steps `env:` property as per https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow.

applies to user tests below as well

Reply:
ah good call! updated in https://github.com/rancher/dashboard/pull/9029/commits/d9e95288448b4fef66b3b8b77c34a8df9c958403